core/src/com/unciv/ui/screens/diplomacyscreen/MajorCivDiplomacyTable.kt

```diff
@@ -85,12 +85,12 @@ class MajorCivDiplomacyTable(private val diplomacyScreen: DiplomacyScreen) {
         if (otherCiv.cities.isNotEmpty() && otherCiv.getCapital() != null && viewingCiv.hasExplored(otherCiv.getCapital()!!.getCenterTile()))
             diplomacyTable.add(diplomacyScreen.getGoToOnMapButton(otherCiv)).row()
 
-        if (!otherCiv.isHuman()) { // human players make their own choices
+//        if (!otherCiv.isHuman()) { // human players make their own choices
```

Comment:
Why this change?

Reply:
Oooops, that slipped through. I was testing if the modifiers worked correctly.